### ISSUE TITLE
Windows class contains window header and footer as constants

### ISSUE THREAD

**Issue by mgrouch** [rjaros/kvision#375]
Not only they are constants they are also private. 

What if our style requires different size of windows header?

Thanks

**Comment by rjaros** [comment]
I've made some small changes. It will be possible to set both header height and the resize handle size in window subclasses. E.g.:
```Kotlin
class Win : Window("Test window", 300.px, 300.px, isDraggable = true, isResizable = true) {
    init {
        header.height = 60.px       // Use header with 60px height
        resizeHandleHeight = 0      // Do not force bottom margin at all
        simplePanel {
            width = 100.perc
            height = 100.perc
        }
    }
}
```

**Comment by rjaros** [comment]
Released with 5.10.0
